quintype/quintype-node-seo
# fix: Amphtml error

# Description

Ref ticket: https://github.com/quintype/ace-planning/issues/656

Fixes # (issue-reference)

Change amphtml url structure depending on the story template.
If its a visual story, amphtml href will be `domain/story-slug` orelse  it will be `domain/amp/story/story-slug`. 
Because visual stories already have `ampstories/` in their storyslug and thats causing the issue. 

## Type of change

Please delete options that are not relevant.



# How Has This Been Tested?

1. Open PrabhatKhabhar [visual story url](https://www.prabhatkhabar.com/entertainment/bhojpuri-actor-khesari-lal-yadav-wife-chanda-beautiful-pics-family-know-about-actor-personal-life-dvy)
2. Open view page source 
3. In the page source, Check for amphtml  url -- it should be https://www.prabhatkhabar.com/ampstories/entertainment/bhojpuri-actor-khesari-lal-yadav-wife-chanda-beautiful-pics-family-know-about-actor-personal-life-dvy

# Checklist:
